packages/rlc-common/src/transformSampleGroups.ts

```diff
@@ -257,6 +272,11 @@ function convertClientLevelParameters(
         "DefaultAzureCredential",
         importedDict
       );
+    } else {
+      clientParams.push({
+        name: "credential",
+        assignment: `const credential = {getToken: () => Promise.resolve({ token: "{Your token}", expiresOnTimestamp: 0 })};`
```

Comment:
@joheredi I wonder if you have any concerns for this part for generating the mock value for token credential as we don't have `@azure/identity` in non-branded scope.